### PR TITLE
feat(meshservice): run tests with meshservice as well

### PR DESCRIPTION
**Pull request by lahabana**

### CODE REVIEW

**Comment by lahabana** [comment]
The tests failed on EKS: https://github.com/Kong/mesh-perf/actions/runs/9516685038/job/26233497943

File: test/k8s/simple_test.go [hunk 1]
```diff
@@ -20,137 +21,147 @@ import (
...
-				Fail(err.Error())
-			case <-time.After(stabilizationSleep):
-				break Loop
+					"--set":          strings.Join(helmFlags, ","),
+					"--env-var":      strings.Join(envVars, ","),
+					"--license-path": kmeshLicense,
+				}),
+			}
+
+			alternativeContainerRegistry, _ = os.LookupEnv("ALTERNATIVE_CONTAINER_REGISTRY")
```
Comment (jijiechen) [inline test/k8s/simple_test.go:58]: May not be relating to this PR, but should we remove this `alternativeContainerRegistry` now? Looks like it's pointing to a personal image repository defaultly. I guess it's an ex-member of the team?

**Review by jijiechen — APPROVED** [review]

**Comment by slonka** [comment]
@lobkovilya can you take a look?
@lahabana can you resolve the conflicts?

**Comment by lahabana** [comment]
I'm lost with this... @jakubdyszkiewicz I remember you talking about this what was the outcome?

**Comment by lahabana** [comment]
Superseded by: https://github.com/Kong/mesh-perf/pull/224

**Pull request closed without merging**